site/book/06-deploying-packages/03-handling-dependencies.md

```diff
@@ -0,0 +1,74 @@
+Sometimes resources within a package have dependencies that require
+one resource to be applied and reconciled before the next resource
+is applied. For example, a package that includes both Wordpress and
+MySQL might require that MySQL is running before Wordpress is started.
+
+In kpt, this is supported by declaring dependencies with the 
+`config.kubernetes.io/depends-on` annotation.
+
+If you look at the `wordpress-with-dependencies` package, a modified
```

Comment:
Provide the `pkg get` command, and walk the user (At no point should the user be left not knowing what to do).